alfresco-integration/src/main/java/com/github/dynamicextensionsalfresco/osgi/spring/WebApplicationSystemPackageFactoryBean.java

```diff
@@ -47,8 +53,16 @@ public boolean isSingleton() {
 	}
 
 	@Override
-	public Set<SystemPackage> getObject() throws Exception {
-		return createSystemPackages();
+	public Set<SystemPackage> getObject() {
+		final long start = System.currentTimeMillis();
+		Set<SystemPackage> systemPackages = createSystemPackages();
+		final long finish = System.currentTimeMillis();
+		if (logger.isInfoEnabled() && (finish - start) > 1500L) {
+			logger.info("System Package scanning took '{}' milliseconds. Possible improvement: enable System Package "
+					+ "cache by adding 'osgi.container.system-package-cache.mode:ENABLE' "
+					+ "to 'osgi-container.properties' (DISABLE by default because of #281)", (finish - start));
```

Comment:
Do not refer to an issue number here, it is irrelevant.
You can add a comment referencing the github issue (full url in that case)